### PR TITLE
TPD-1702, TPD-1715 ManageGiving OneTimeLink Fix

### PR DESCRIPTION
**Pull request by dahnbaez**
https://touchpointsoftware.atlassian.net/browse/TPD-1702?atlOrigin=eyJpIjoiOTFmMzczZjI2ZWZkNDQyYWFjNWYwYjQ5NGNjNTdlNTQiLCJwIjoiaiJ9
[touchpointsoftware.atlassian.net/browse/TPD-1715](https://touchpointsoftware.atlassian.net/browse/TPD-1715)

### CODE REVIEW

File: CmsWeb/Areas/OnlineReg/Models/OnlineReg/CreateAccount.cs [hunk 1]
```diff
@@ -45,12 +45,16 @@ public User CreateAccount()
...
         {
+            if (PeopleId != null)
+            {
+                peopleId = PeopleId;
+            }           
             var ot = new OneTimeLink
             {
                 Id = Guid.NewGuid(),
-                Querystring = $"{divid ?? orgid ?? masterorgid},{PeopleId}"
+                Querystring = $"{divid ?? orgid ?? masterorgid},{peopleId}"
```
Comment (josearmandohf) [inline CmsWeb/Areas/OnlineReg/Models/OnlineReg/CreateAccount.cs:57]: Have you tested the variable Querystring when sending peopleID as null? 
like Querystring = = $"{divid ?? orgid ?? masterorgid},{**NULL**}"

File: UnitTests/CMSWebTests/Areas/OnlineReg/Models/OnlineReg/OnlineRegPersonModelTests.cs [hunk 2]
```diff
@@ -39,7 +39,7 @@ public void SendOneTimeLinkTest()
                 orgid = db.Organizations.Select(o => o.OrganizationId).Max(),
                 PeopleId = person.PeopleId
             };
-            model.SendOneTimeLink("admin@example.com", "/OnlineReg/ManageGiving/", "this is a test", "{first}, this is only a test {url}");
+            model.SendOneTimeLink("admin@example.com", "/OnlineReg/ManageGiving/", "this is a test", "{first}, this is only a test {url}", peopleId : person.PeopleId);
```
Comment (josearmandohf) [inline UnitTests/CMSWebTests/Areas/OnlineReg/Models/OnlineReg/OnlineRegPersonModelTests.cs:42]: can you add after line 42 a model with peopleID = NULL?

**Review by josearmandohf — CHANGES_REQUESTED** [review]
Can you please add in the test what happened if PeopleID equals null?

**Review by brondavies — APPROVED** [review]
